### DISCUSSION TITLE
Scores way higher than the 1,000,000 max score limit

### DISCUSSION THREAD

**Discussion by Zyflx** [ppy/osu#16991] [Q&A] [answered]
Not sure if this has happened to anyone else on lazer(i’m on iOS)but for some reason my scores are way higher than what they should be this started happening a few days ago.There is a image below to represent what I’m talking about
![A42CEF06-4D60-4C9A-A47C-FF6CC5A894D8](https://user-images.githubusercontent.com/100444522/155809096-83660fc0-6e27-42da-b639-d4a5f372b3a3.png)

**Comment by Theighlin** [comment, marked as answer]
You must've switched the scoring display setting to "Classic" instead of "Standardised" in the settings. It's intended that you can view scores the way you prefer to.

**Reply by Zyflx** [reply]
Ok got it
